samples/name_domains_values/Coded_Domain_HostileOrEnemy.csv

```diff
@@ -0,0 +1,2 @@
+Name,Value
+Hostile (Enemy),ENY
```

Comment:
I'm not sure we will want to have a field/domain for this is the GDB Control Measure feature classes (there is not one not currently one anyway) - so let's chat if/before you add this field/domain to the GDB since it is not currently a user-driven text field like the other text amplifiers (it is kind of an odd case).